rest/example/options/snapshots-assets/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+
+	polygon "github.com/polygon-io/client-go/rest"
+	"github.com/polygon-io/client-go/rest/models"
+)
+
+func main() {
+	// Init client
+	c := polygon.New(os.Getenv("POLYGON_API_KEY"))
+
+	// Set parameters
+	params := models.ListAssetSnapshotsParams{}.
+		WithTickerAnyOf("AAPL,META,F")
```

Comment:
since this example is in the options directory, probably makes more sense to make these options tickers

Reply:
Ah, good catch! Thanks :)